src/App.js

```diff
@@ -1,41 +1,22 @@
 import "./App.css";
-import logo from "./logo.png";
+import {Route, Routes} from "react-router-dom";
+import {LandingPage} from "../src/pages/LandingPage/LandingPage";
+import {Navbar} from "../src/components/Navbar/Navbar";
+import { CategoryFilterPage } from "./pages/CategoryFilterPage/CategoryFilterPage";

```

Comment:
Please add a formatter like prettier. It will format your code which will increase code readability. Also use absolute imports to avoid  ../../